.travis.yml

```diff
@@ -3,6 +3,7 @@ python: 3.6
 
 env:
   global:
+  - ORG_NAME: "coala"
```

Comment:
Not allowed to put the word coala anywhere in this repo; it must work dynamically; when forked to a different org, it must derive the org.  I've also now created https://github.com/coala/community/issues/29 for the existing use of `coala` in this file.